code/datums/gamemode/role/vampire_role.dm

```diff
@@ -386,7 +350,8 @@
 -- Helpers --
 */
 
-/datum/role/vampire/proc/update_vamp_hud()
+// ! - need to handle thrall removing
+/datum/role/vampire/proc/update_vamp_hud(var/datum/role/thrall/new_T = null)
```

Comment:
AAAA IT'S DEPRECIATED, STOP

Reply:
![image](https://user-images.githubusercontent.com/6307265/38877943-20c196a0-4260-11e8-8b8f-85e6829ae4a1.png)